### ISSUE TITLE
The str accessor does not work on columns selected using the .columns property

### ISSUE THREAD

**Issue by risenW** [opensource9ja/danfojs#181]
Selecting a column from a dataframe returns a Series, but the `.str` accessor does not work when column has NaN values. For example running the following:
```javascript
data = { "Name": ["Apples", "App", "Banana", undefined],
            "Count": [NaN, 5, NaN, 10] ,
            "Price": [200, 300, 40, 250] }

df = new dfd.DataFrame(data)
name_sf = df.column("Name")
name_sf.str.startsWith("App")

```
gives the following error:
```bash
TypeError: n.startsWith is not a function

```

**Comment by adithyaakrishna** [comment]
@risenW Ig this can be fixed by using name.sf.str.startsWith(data.Name[1]). Not sure tho, as I'm still learning 😶

**Comment by risenW** [comment]
> @risenW Ig this can be fixed by using name.sf.str.startsWith(data.Name[1]). Not sure tho, as I'm still learning 😶

Note really. I think the problem is caused by the fact that the value is not cast to a String type. So therefore calling string functions on it do not work. 

Try casting the values to string explicitly first. I'm sure this will be fixed in the [str class. ](https://github.com/opensource9ja/danfojs/blob/master/danfojs-node/src/core/strings.js)

**Comment by adithyaakrishna** [comment]
Got it, will try that 💯

**Comment by risenW** [comment]
FIXED in #213
